blender/LilySurfaceScrapper/CyclesMaterialData.py

```diff
@@ -40,59 +39,66 @@ def createMaterial(self):
         links = mat.node_tree.links
         principled_mat = PrincipledBSDFWrapper(mat, is_readonly=False)
         principled = principled_mat.node_principled_bsdf
-        back_principled = None
         mat_output = principled_mat.node_out
         principled_mat.roughness = 1.0
-        normal_node = None
-        displacement_node = None
+        front = {}
+        back = {}
 
+        # Create all of the texture nodes
         for map_name, img in self.maps.items():
             if img is None or map_name.split("_")[0] not in __class__.input_tr:
                 continue
-
-            current_principled = principled
-            if map_name.endswith("_back"):
-                if back_principled is None:
-                    # Create backface principled and connect it
-                    back_principled = nodes.new(type="ShaderNodeBsdfPrincipled")
-                    geometry_node = nodes.new(type="ShaderNodeNewGeometry")
-                    mix_node = nodes.new(type="ShaderNodeMixShader")
-                    back_principled.inputs["Roughness"].default_value = 1.0
-                    links.new(geometry_node.outputs["Backfacing"], mix_node.inputs[0])
-                    links.new(principled.outputs[0], mix_node.inputs[1])
-                    links.new(back_principled.outputs[0], mix_node.inputs[2])
-                    links.new(mix_node.outputs[0], mat_output.inputs[0])
-                current_principled = back_principled
-                map_name = map_name[:-5]  # remove "_back"
-                
+            
             texture_node = nodes.new(type="ShaderNodeTexImage")
+            if "_back" in map_name:
+                map_name = map_name[:-5] # remove "_back"
+                back[map_name] = texture_node
+            else:
+                front[map_name] = texture_node
+            
             texture_node.image = getCyclesImage(img)
             texture_node.image.colorspace_settings.name = "sRGB" if map_name == "baseColor" else "Non-Color"
-
             if hasattr(texture_node, "color_space"):
                 texture_node.color_space = "COLOR" if map_name == "baseColor" else "NONE"
-            elif map_name == "glossiness":
+            if map_name == "opacity":
+                mat.blend_method = 'BLEND'
+
+        front = back if not front else front # In case just the backside texture was chosen
```

Comment:
```suggestion
        if not front:
            front, back = back, front
```
It is important that `back` gets null if there is only data in back `if not back` a little later will not get triggered.

Reply:
https://github.com/ogierm/LilySurfaceScrapper/commit/c59603ccabf8f3f4e3d9e19862b8537f4a84d89d#r34949751